Assets/_Project/Code/Scripts/QuestionStage/AnswerControl.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using Unity.Netcode;
+using System.Linq;
+public class AnswerControl : NetworkBehaviour
+{
+    public TMP_InputField answerInput;
+    public TMP_Text questionText;
+    public TMP_Text feedbackText;
+    public TMP_Text timerText;
+    public Button submitButton;
+    public TMP_Text numerRundy;
+    public TMP_Text buttonText1, buttonText2, buttonText3, buttonText4;
+    public Button answerButton1, answerButton2, answerButton3, answerButton4;
+    public Button hintButton;
+
+    public static ulong IdOfPlayerWhoWonAuction = 0;
```

Comment:
1. nazwa zmiennej może być skrócona do winnerId, 
2. nazwy pól piszemyu z małej litery, 
3. to powinien być technicznie obiekt, bo rpc pozwala przekazywać obiekty między clientów